### ISSUE TITLE
Add Bluetooth COM Port supports for Seika v3/v5/v80 Braille display

### ISSUE THREAD

**Issue by moyanming** [nvaccess/nvda#14242]
Seika v3/v5/v80 have a CP2102 chip (USB-To-Uart) and a Bluetooth, the CP2102 works well with NVDA now, but the Bluetooth COM Port cannot be selected when using Bluetooth connection.
The Seika v3/v5/v80 use Bluetooth name: "TS5 abcd", where "abcd" are four digital numbers.

[#13191](https://github.com/nvaccess/nvda/pull/13191)
[#13142](https://github.com/nvaccess/nvda/issues/13142)

### Steps to reproduce:
select: NVDA->Braille->"Seika v3/v5/v80" item, but cannot select Bluetooth COM Port.

### Expected behavior:
There should have COM port can be selected when using Bluetooth connection.

#### NVDA version:
NVDA 2022.3

#### Windows version:
Windows 10 21H2

**Comment by Adriani90** [comment]
@moyanming, @school510587 is this still an issue with NVDA 2025.1 Beta and Seika braille displays?

**Comment by school510587** [comment]
Hi,

Sorry, I have no chance to use any Seika braille display recently. Thus, I cannot answer the problem.
